test/dispatchers/touchDispatcherTests.ts

```diff
@@ -2,249 +2,282 @@
+        touchDispatcher.onTouchMove(callback);
+        TestMethods.triggerFakeTouchEvent("touchmove", svg, expectedPoints, ids);
+        assert.isTrue(callbackWasCalled, "callback was called on touchmove");
 
-      target.remove();
-      callbackWasCalled = false;
-      TestMethods.triggerFakeTouchEvent("touchmove", target, expectedPoints, ids);
-      assert.isFalse(callbackWasCalled, "callback was not called after <svg> was removed from DOM");
+        svg.remove();
+        callbackWasCalled = false;
+        TestMethods.triggerFakeTouchEvent("touchmove", svg, expectedPoints, ids);
+        assert.isFalse(callbackWasCalled, "callback was not called after <svg> was removed from DOM");
 
-      td.offTouchMove(callback);
-    });
+        touchDispatcher.offTouchMove(callback);
+      });
 
-    it("doesn't call callbacks if obscured by overlay", () => {
-      let targetWidth = 400, targetHeight = 400;
-      let target = TestMethods.generateSVG(targetWidth, targetHeight);
-      // HACKHACK: PhantomJS can't measure SVGs unless they have something in them occupying space
-      target.append("rect").attr("width", targetWidth).attr("height", targetHeight);
-
-      let targetXs = [17, 18, 12, 23, 44];
-      let targetYs = [77, 78, 52, 43, 14];
-      let expectedPoints = targetXs.map((targetX, i) => {
-        return {
-          x: targetX,
-          y: targetYs[i]
-        };
+      it("doesn't call callbacks if obscured by overlay", () => {
+        let targetXs = [17, 18, 12, 23, 44];
+        let targetYs = [77, 78, 52, 43, 14];
+        let expectedPoints = targetXs.map((targetX, i) => {
+          return {
+            x: targetX,
+            y: targetYs[i]
+          };
+        });
+        let ids = targetXs.map((targetX, i) => i);
+
+        let callbackWasCalled = false;
+        let callback = () => callbackWasCalled = true;
+
+        touchDispatcher.onTouchStart(callback);
+        TestMethods.triggerFakeTouchEvent("touchstart", svg, expectedPoints, ids);
+        assert.isTrue(callbackWasCalled, "callback was called on touchstart");
+
+        let element = <HTMLElement> svg[0][0];
+        let position = { x: 0, y: 0 };
+        while (element != null) {
+          position.x += (element.offsetLeft || element.clientLeft || 0);
+          position.y += (element.offsetTop || element.clientTop || 0);
+          element = <HTMLElement> (element.offsetParent || element.parentNode);
+        }
+
+        let overlay = TestMethods.getSVGParent().append("div")
+              .style({
+                height: "400px",
+                width: "400px",
+                position: "absolute",
+                top: position.y + "px",
+                left: position.x + "px"
+              });
+
+        callbackWasCalled = false;
+        TestMethods.triggerFakeTouchEvent("touchmove", svg, expectedPoints, ids);
+        assert.isFalse(callbackWasCalled, "callback was not called on touchstart on overlay");
```

Comment:
This method and the preceding line don't match up -- are we checking `touchmove` or `touchstart`?